app/services/collators/childcare_collator.rb

```diff
@@ -31,7 +30,7 @@ def monthly_child_care_cash
     end
 
     def applicant_has_dependant_child?
```

Comment:
Oh, this should be `person_has_dependant_child?` and likewise `applicant_is_employed?` and `applicant_has_student_loan?` should be renamed